godot-macros/src/class/data_models/func.rs

```diff
@@ -59,18 +59,20 @@ pub fn make_method_registration(
     class_name: &Ident,
     func_definition: FuncDefinition,
 ) -> TokenStream {
-    let signature_info = get_signature_info(&func_definition.func, func_definition.has_gd_self);
+    let signature_info = into_signature_info(
+        class_name,
+        func_definition.func,
+        func_definition.has_gd_self,
+    );
     let sig_tuple =
         util::make_signature_tuple_type(&signature_info.ret_type, &signature_info.param_types);
 
-    let method_name = &signature_info.method_name;
-    let param_idents = &signature_info.param_idents;
```

Comment:
didn't really need a reference binding to param_idents when we only use it to iterate over, especially now that it's owned.

Just moved the method name down to where the references are used a couple times so it's obvious why we're still taking a reference.